frontends/common/constantParsing.cpp

```diff
@@ -77,7 +77,7 @@ int parseConstantChecked(const Util::SourceInfo &srcInfo, const UnparsedConstant
     auto cst = parseConstant(srcInfo, constant, 0);
     if (!cst->fitsInt()) {
         ::P4::error(ErrorType::ERR_OVERLIMIT,
-                    "%1$x: this implementation does not support bitstrings this large", cst);
+                    "%1%: this implementation does not support bitstrings this large", cst);
```

Comment:
This changes from printing the bitstring a hex to printing it as decimal, which seems less useful.  Is there a way to have boost::formatter print in hex?

Reply:
`%x` should give hex printing if using the abseil formatter. Not quite clear how to migrate smoothly here...

The problem here in particular is that cst is a constant and the `<<` doesn't use hex.  And, according to the reference files, this output already generated decimal format: https://github.com/p4lang/p4c/blob/main/testdata/p4_16_errors_outputs/width_e.p4-stderr#L1